### PR TITLE
Expenses Redesign

### PR DESCRIPTION
**Pull request by carlosascari**
For https://github.com/OpenCollective/OpenCollective/issues/165

preview

Donations has also been affected by the redesign

preview

Related:
https://github.com/OpenCollective/OpenCollective/issues/174

### CODE REVIEW

**Comment by asood123** [comment]
How does it scale on phone screens?

File: frontend/src/components/ExpenseForm.js [hunk 1]
```diff
@@ -137,32 +112,20 @@ class ExpenseForm extends Component {
...
-              value={attributes.notes}
-              handleChange={notes => appendExpenseForm({notes})} />
-          </div>
-
-          <div className="buttonsRow">
-            <SubmitButton label={i18n.getString('submit')} />
-            <Button color="red" label={i18n.getString('cancel')} onClick={this.onCancel.bind(this)} />
+            
+          <div className='col col-12 sm-col-12 md-col-6 lg-col-6  pl1'>
+            <label>PayPal account/ Account number</label>
```
Comment (asood123) [inline frontend/src/components/ExpenseForm.js]: I'd s/Paypal Account/Paypal Email. What's the "Account number"?

File: frontend/src/components/ExpenseForm.js [hunk 2]
```diff
@@ -42,93 +31,79 @@ class ExpenseForm extends Component {
...
-            <Input
-              customClass='js-transaction-description'
-              hasError={expense.error.title}
-              value={attributes.title}
-              handleChange={title => appendExpenseForm({title})} />
+        <Notification {...this.props} autoclose/>
+        <div className='line1'>expense details</div>
+        <div className='clearfix input-container'>
+          <div className='col col-12 sm-col-12 md-col-6 lg-col-6 pr1'>
+            <ImageUpload
```
Comment (asood123) [inline frontend/src/components/ExpenseForm.js]: can you make this a required field? there's an issue already open on that

**Comment by asood123** [comment]
Tested it on mobile - looks great. Can you add a link "Submit another expense" after the expense is submitted.

**Comment by asood123** [comment]
When the user is logged, it's not pre-filling any of the fields. It should prefill email (and not let you edit it). Everything else is fine to let you edit.

File: frontend/src/components/ExpenseForm.js (hunk 2 above)
Comment (carlosascari) [inline frontend/src/components/ExpenseForm.js]: The submit button is disabled until it an image is uploaded

File: frontend/src/components/ExpenseForm.js (hunk 1 above)
Comment (carlosascari) [inline frontend/src/components/ExpenseForm.js]: It will be removed, so it will only say: `PayPal Email`

**Pull request merged**